element_array_ephys/ephys_no_curation.py

```diff
@@ -763,8 +763,19 @@ def make(self, key):
                         "spike_depths": spike_depths,
                     }
                 )
-        else:  # read from kilosort outputs
-            raise NotImplementedError
+        elif (
+            not si_sorting_analyzer_dir.exists()
+            and (output_dir / sorter_name / "spikeinterface_report").exists()
+        ):
+            # Corresponds to a version of spikeinterface == 0.100.* since `sorting_analyzer` was introduced in version 0.101.*
+            raise NotImplementedError(
+                "Old version of SpikeInterface (== 0.100.*) not supported."
+            )
```

Comment:
I'm not sure about this check and raise error here.
`(output_dir / sorter_name / "spikeinterface_report")` is not a SpikeInterface 0.100.* thing, it is our own implementation

I'm afraid this `elif` logic is not going to be future proof, and added to handle a one-time issue that we created for ourselves for the organoids project

Reply:
I agree. Let's consider that `NotImplementedError` exceptions related to an earlier version of `SpikeInterface`( and not involving the use of `Kilosoft`) will be isolated cases and can be ignored. So the logic remains unmodified ([revert changes in CuratedClustering](https://github.com/dj-sciops/utah_organoids_element-array-ephys/pull/49/commits/35a892e7276b3b7147d0b1ef7372de63a38c98ce)). Thanks for the comment.